src/app/Board/index.ts

```diff
@@ -34,7 +34,21 @@ export class Board implements Renderable {
       this.tiles[x][y] = tile;
       this.rows[x].append(tile.container);
     })
+    this.initializeOrganisms()
+    this.render();
+  }
+
+  private initializeOrganisms() {
+    if (INITIAL_ORGANISMS_NUMBER > this.xSize * this.ySize) {
```

Comment:
Does `INITIAL_ORGANISMS_NUMBER` include the player as well? If no, you have to subtract `1` here